h/views/client.py

```diff
@@ -49,6 +49,16 @@ def sidebar_app(request, extra=None):
     app_config = {
         'apiUrl': request.route_url('api.index'),
         'authDomain': request.authority,
+        'oauthClientId': settings.get('h.client_oauth_id'),
+
+        # The OAuth feature flag is included as part of the `app.html` config
+        # rather than being delivered via the "features" key in /api/profile so
+        # that it is available as soon as the client starts before
+        # API tokens are fetched.
+        #
+        # This should be removed once OAuth for first-party accounts is shipped.
```

Comment:
I'd put `# TODO` here (has been done with other comments like this in h, you can grep it)